mbed-os-env-cmake/Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN pip3 install -U \
 
 # Set up mbed environment
 WORKDIR /root/
-RUN wget https://github.com/ARMmbed/mbed-os/raw/master/requirements.txt && \
+RUN wget https://github.com/ARMmbed/mbed-os/raw/master/tools/cmake/requirements.txt && \
```

Comment:
```suggestion
RUN wget https://github.com/ARMmbed/mbed-os/raw/latest/tools/cmake/requirements.txt && \
```
@0xc0170 @donatieng What do you think? Or is `latest` not recommended and for backward compatibility of the old tools only?

Reply:
latest would be an answer to have released version always. But then a question is what tools we should install ? We fetch any the latest (major versions could break the build).

Reply:
github.com/ARMmbed/mbed-os-docker-images/pull/6 does also to latest :-)

Reply:
I agree with @Patater 

I will do that as part of https://github.com/ARMmbed/mbed-os-docker-images/pull/7 Once done, I will close this PR.

@pan- @donatieng @0xc0170  please shout if that's not alright.